charts/xapp-hello-world/values.yaml

```
@@ -80,6 +78,12 @@ securityContext:
   # readOnlyRootFilesystem: true
   runAsNonRoot: true
   runAsUser: 1000
+  runAsGroup: 1000
+
+persistence:
+  storageClass: "local-path"
```

We should leave this empty to use the default storage class.

```suggestion
  storageClass: ""
```